### PR TITLE
Sporting Shotgun (code) references removed 

### PR DESCRIPTION
**Pull request by nahmed99**

### CODE REVIEW

**Review by saruniitr — COMMENTED** [review]
There are many unrelated formatting issues. This could be mainly because your `black` version is different. Please use the correct version to not make these changes.
Please check with @Pithikos , he had similar problem and corrected it now.

**Review by eadpearce — APPROVED** [review]
lgtm

File: api/goods/helpers.py [hunk 1]
```diff
@@ -201,7 +190,7 @@ def check_if_unsupported_fields_edited_on_firearm_good(data):
 
 def has_valid_certificate(organisation_id, document_type):
     certificate_exists = DocumentOnOrganisation.objects.filter(
-        organisation=organisation_id, document_type=document_type,
+        organisation=organisation_id, document_type=document_type
```
Comment (saruniitr) [inline api/goods/helpers.py:193]: these unrelated black formatting changes need to be reverted

**Review by saruniitr — CHANGES_REQUESTED** [review]
Please remove unrelated black formatting changes, there are many to ignore.

File: api/goods/helpers.py (hunk 1 above)
Comment (nahmed99) [reply api/goods/helpers.py]: Done.

**Review by saruniitr — APPROVED** [review]
ty, looks good to me now

**Pull request merged**